megamek/src/megamek/common/Hex.java

```diff
@@ -426,6 +426,19 @@ public int terrainLevel(int type) {
     public ITerrain getTerrain(int type) {
         return terrains.get(type);
     }
+    
+    @Override
+    public ITerrain getAnyTerrainOf(int type, int... types) {
```

Comment:
Why not just `int... types` ?

Reply:
To make the call getAnyTerrainOf() invalid. (Enforce giving at least one parameter)
Which I thought int... doesnt do, or does it?

Reply:
Actually I wanted to comment on this in the PR description but of course forgot. It might seem useless but I find this comes up pretty often; looking in a hex for any terrain of (x,y,z) and then getting whichever of those is present, where x,y,z are usually mutually exclusive. This is usually code-heavy and ugly so I wanted to simplify this.

Reply:
Actually, I really enjoy your usage of the API calling convention itself to enforce At Least One semantics. Elegant and I missed that.

Definite Pit of Success!